src/kiota/KiotaPluginCommands.cs

```diff
@@ -37,6 +37,28 @@ internal static Option<List<PluginType>> GetPluginTypeOption(bool isRequired = t
         typeOption.AddValidator(x => KiotaHost.ValidateKnownValues(x, "type", Enum.GetNames<PluginType>()));
         return typeOption;
     }
+
+    internal static Option<PluginAuthType> GetPluginAuthTypeOption(bool isRequired = false)
+    {
+        var authTypeOption = new Option<PluginAuthType>("--auth-type", "The authentication type for the plugin. Should be a valid OpenAPI security scheme.");
+        authTypeOption.AddAlias("-at");
+        {
+            authTypeOption.IsRequired = isRequired;
+            authTypeOption.Arity = ArgumentArity.ZeroOrOne;
+        }
+        authTypeOption.AddValidator(x => KiotaHost.ValidateKnownValues(x, "auth-type", Enum.GetNames<PluginAuthType>()));
+        return authTypeOption;
+    }
+
+    internal static Option<string> GetPluginAuthRefIdOption(bool required = false)
+    {
+        var authRefIdOption = new Option<string>("--auth-ref-id", "The authentication reference id for the plugin.")
+        {
+            IsRequired = required,
+        };
+        authRefIdOption.AddAlias("--arid");
```

Comment:
I don't know how useful this is

Reply:
I'd suggest we bring the full parameter name and the alias decision to the PM @maisarissi  and this is an experience point.
"auth" as a full parameter name is an abbreviation, which is generally avoided for full form.
And yes, this alias is a bit confusing.

Reply:
Ok, I'll remove the aliases for now until we have a clear direction and update the relevant specs.

Reply:
So, I though about the full form to be --auth-type and alias to --auth, and for --auth-ref-id, --refid.
Any concerns about that?

Reply:
what about --authentication-type and --authentication-reference-id ?

Reply:
Going with this for now:
| Full form  | Alias |
| ------------- |:-------------:|
| --authentication-type      | --at     |
| --authentication-reference-id      | --refid     |